files/brooklyn.properties

```diff
@@ -252,6 +252,10 @@ brooklyn.location.jclouds.minRam = 2048
 # brooklyn.location.named.On-Prem\ Iron\ Example.privateKeyFile=~/.ssh/produser_id_rsa
 # brooklyn.location.named.On-Prem\ Iron\ Example.privateKeyPassphrase=s3cr3tpassphrase
 
+brooklyn.location.named.byon=byon:(hosts="10.10.10.101,10.10.10.102,10.10.10.103,10.10.10.104,10.10.10.105")
+brooklyn.location.named.byon.user=vagrant
+brooklyn.location.named.byon.password=vagrant
```

Comment:
I would rather call it `byon-cluster` to avoid confusion with the single byon nodes